libscanbuild/__init__.py

```diff
@@ -14,6 +14,7 @@
 import shlex
 import subprocess
 import sys
+from pprint import pformat
```

Comment:
Please follow the pattern to be consistent. Standard packages are imported without `from` keyword.